validate.lic

```diff
@@ -503,6 +503,18 @@ class DRYamlValidator
     warn("Your hometown of '#{settings.hometown}' is not valid. Valid values are: #{valid_hometowns}")
   end
 
+  def assert_that_force_healer_town_is_valid(settings)
+    return unless settings.force_healer_town
+
+    valid_hometowns = get_data('town')
+                      .to_h
+                      .keys
+                      .map(&:to_s)
+
+    return if valid_hometowns.include? settings.force_healer_town
+    warn("Your force_healer_town setting of '#{settings.force_healer_town}' is not valid. Valid values are: #{valid_hometowns}")
```

Comment:
Did you consider checking that the hometown also has an empath set?

Reply:
Done, and tested that it works.

Reply:
Should this be `error` instead of `warn`?